issuebot

```diff
@@ -131,6 +168,12 @@ lilac can't parse out the relevant package names, please handle manually.''')
     with file_lock(LILAC_LOCK):
       subprocess.check_output(['git', 'pull'], cwd=REPO)
 
+      for name in os.listdir(REPO):
+        if not os.path.isdir(os.path.join(REPO, name)):
+            continue
```

Comment:
I've added a `lilac2.lilacyaml.iter_pkgdir` function to iterate through all package directories.

Reply:
Thanks! I moved to that in the latest commit.